TodoApp/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "demo",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack-dev-server --mode development --open --hot",
+    "test": "jest --watchAll",
+    "build": "webpack --mode production",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix",
+    "format": "prettier --write './**/*.{js,jsx,ts,tsx,css,md,json}' --config ./.prettierrc"
```

Comment:
`format` command will fail since `.prettierrc` file is not present.
Fix and format your entire codebase.